src/main/webapp/script.js

```diff
@@ -46,6 +46,21 @@ function createEntryElement(entry) {
     console.log(entry.score);
   });
 
+  const sentimentIconElement = document.createElement('img');
+//   sentimentIconElement.height = '4vw';
+//   sentimentIconElement.width = '4vw';
+  sentimentIconElement.className = 'sentimentIcon';
+  if (entry.score < -0.2) {
```

Comment:
I think I do <= -0.2 for the chart (instead of entry.score < -0.2), so for the sake of consistency I think we should change this here?

Reply:
did you mean to write a different number?